sys/net/network_layer/ng_icmpv6/ng_icmpv6.c

```diff
@@ -62,10 +62,17 @@ void ng_icmpv6_demux(kernel_pid_t iface, ng_pktsnip_t *pkt)
 
     LL_SEARCH_SCALAR(pkt, icmpv6, type, NG_NETTYPE_ICMPV6);
 
+    if (icmpv6 == NULL) {
```

Comment:
This should also be in here (otherwise the function isn't called) => use `assert()` here too